lib/Amin/Machine/Machine_Spec/machine_spec.xml

```diff
@@ -1,11 +1,12 @@
 <machine xmlns:amin="http://projectamin.org/ns/">
 	<!--	LICENSE:
 	        Please see the LICENSE file included with this distribution 
-		or see the following website http://projectamin.org.
 	-->
 	<!--what type of machine is this? -->
 	<name>Amin::Machine::Dispatcher</name>	
-	<!-- <amin:command name="mkdir"/> -->
```

Comment:
Are we going to be adding such for all filters in machine spec?

Reply:
no i thought i took em all out :)

Reply:
No worries can clean up next commit :)